packages/components/Shimmer/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@fluentui-react-native/shimmer",
+  "version": "0.1.0",
+  "description": "A shimmer view",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
```

Comment:
I'm curious whether this should also have "lint" and "test" scripts, at the very least the linting one seems like a good idea. For jest you can look at how some of the other components are doing snapshot testing and emulate that.